sdk_extensions/zpages/src/test/java/io/opentelemetry/sdk/extensions/zpages/TracezZPageHandlerTest.java

```diff
@@ -274,18 +274,94 @@ public void summaryTable_linkForErrorSpans() {
   }
 
   @Test
-  public void spanDetails_emitSpanNameCorrectly() {
+  public void spanDetails_emitRunningSpanDetailsCorrectly() {
+    OutputStream output = new ByteArrayOutputStream();
+    Map<String, String> queryMap = new HashMap<String, String>();
+    Span runningSpan = tracer.spanBuilder(RUNNING_SPAN).startSpan();
+
+    queryMap.put("zspanname", RUNNING_SPAN);
```

Comment:
Not a big deal but it's a bit less verbose to use `ImmutableMap.of`, it's very close in feeling to a map literal.